src/GDGUkraine/lib/forms/registration_forms.py

```diff
@@ -0,0 +1,268 @@
+import logging
+
+from wtforms.form import Form
+from wtforms.fields import (
+    StringField, TextAreaField, SelectField,
+    BooleanField, SelectMultipleField,
+)
+from wtforms.widgets import ListWidget, RadioInput, CheckboxInput
+from wtforms.validators import Required, Length, Email, URL, Optional
+
+from GDGUkraine.model import (
+    EXPERIENCE_CHOICES, ENGLISH_CHOICES,
+    GENDER_CHOICES, TSHIRT_CHOICES,
+)
+
+from .widgets import InlineWidget
+
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationForm(Form):
+
+    name = StringField(
```

Comment:
I do not like this idea. Having one more level of abstraction (and bugs) for what? Just using yaml?

Also there are a lot of people familiar with WTForms, but not so many, who will be familiar with our own DSL.

KISS and do not create unnecessary entities.